src/bridge/models/bridge.model.ts

```diff
@@ -22,9 +22,19 @@ export interface ApproveData {
   spender: string;
 }
 
-export interface GetTokenBalanceData {
-  tokenAddress: string;
+export interface GetTokenBalanceParams {
+  /**
+   *  The address for which we will find out the token balance
+   */
   account: string;
+  /**
+   *  The token address itself
+   */
+  tokenAddress: string;
+  /**
+   *  If present, the result will be recalculated in precision with token decimals
+   */
+  tokenDecimals?: number;
```

Comment:
Accepting tokenDecimals is not consistent with the other methods in this SDK, which accept either TokenInfo or chain symbol along with the token address to deduce token decimals from that. What do you think?

Reply:
I think it's too overloaded to pass the whole TokenInfo to get only 2 fields, given that you need to add some additional boolean to split with/ without precision

it's pretty simple to just map 1 or 2 property from token info
 const tokenBalanceWithPrecisionData = {
    account: accountAddress,
    tokenAddress: tokenInfo.tokenAddress,
    tokenDecimals: tokenInfo.decimals//optional
  };

Reply:
why is it preferable that the user, in case when he is has a tokenInfo object, would parse it himself, and not the sdk get all the necessary fields?
And what about the case when user doesn't have a tokenInfo object and just wan't to get balance of some token on some chain? Other functions in such case require only chain symbol and token address for the conversion to happen. 
I think when choosing a set of parameters for different methods in the same SDK should follow the same logic.